hivemq-edge/src/frontend/src/modules/Workspace/components/nodes/NodeAdapter.spec.cy.tsx

```diff
@@ -106,13 +106,39 @@ describe('NodeAdapter', () => {
     cy.get('[role="toolbar"] button').eq(2).should('have.attr', 'aria-label', 'Group the selected adapters')
   })
 
-  it('should render the toolbar properly', () => {
+  it.only('should render the toolbar properly', () => {
```

Comment:
was this `only` left on purpose because other tests are failing?

Reply:
Nope. And there are three others!
I'll need an `eslint` rule for that one